src/services/mongoose/accounts.ts

```diff
@@ -36,7 +36,7 @@ export const AccountsRepository = (): IAccountsRepository => {
     walletId: WalletId,
   ): Promise<Account | RepositoryError> => {
     try {
-      const result: UserType = await User.findOne({ walletId }, projection)
+      const result: UserType = await User.findOne({ walletIds: walletId }, projection)
```

Comment:
please rename to `walletId` param or rename the method to use list prefix and pass an array

Reply:
I don't understand this comment:

> please rename to walletId param

we want to get one `walletId` out of the arrays of `walletIds`

> rename the method to use list prefix and pass an array

can you provide an example if you think my syntax is not correct? (it seems to be working at least)

Reply:
Worked ok for me on testing. Looks like Mongoose is able to use `walletIds: walletId` to search for the `walletId` inside an array and return the first object it finds that has an array with the target `walletId`.

Reply:
yup that is also my understanding @vindard